x-pack/plugins/cases/public/plugin.ts

```diff
@@ -51,6 +53,10 @@ export class CasesUiPlugin implements Plugin<void, CasesUiStart, SetupPlugins, S
       hooks: {
         getUseCasesAddToNewCaseFlyout: useCasesAddToNewCaseFlyout,
         getUseCasesAddToExistingCaseModal: useCasesAddToExistingCaseModal,
+        useCasesToast,
```

Comment:
I do not think we should export that hook. The toast should be handled automatically from the `useCasesAddToNewCaseFlyout` and `useCasesAddToExistingCaseModal` hooks. Consumers should not care about this implementation detail (showing a toast when attaching a case). I can see plugins that use cases to copy-pasting the same code (showing the toasters).

Reply:
I agree very much about this, but I was not sure if doing it directly or not. Should I go ahead and do it?

Reply:
Yes please!

Reply:
Changes done in the last commits.